pkg/controller/devModel.go

```diff
@@ -44,10 +44,13 @@ func (ctx *DevModel) GetNetNoDHCPID() string {
 
 //SetWiFiParams set ssid and psk for RPI
 func (ctx *DevModel) SetWiFiParams(ssid string, psk string) {
+	if ctx.devModelType != defaults.DefaultRPIModel {
```

Comment:
Why is this only enabled for RPI? Ultimately this leads to generation of DevicePortConfig and it seems orthogonal to RPi -- it is just is a useful piece of functionality to have.

Reply:
Done. I have added more common variant into this function.